apps/mull-ui/src/app/components/chat-bubble/chat-bubble.scss

```
@@ -0,0 +1,49 @@
+@import '../../../variables.scss';
+
+.chat-container {
+  margin: 1rem 0;
+}
+
+.chat-bubble {
+  max-width: 70%;
+}
+
+.host-chat-container {
```

This chat bubble is going to use in a bunch of places. The `host-chat-container` tag refers to when a chat bubble is made by the current user (just like on Messenger, when you type bubbles then show on the right like for `host-chat-container`). I think the name should be more generic.

```suggestion
.right-chat-container {
```
or 
```suggestion
.current-user-chat-container {
```

This comment applies to all the places you're using `host` and `participant` in this component so that I don't repeat